distribution/pom.xml

```diff
@@ -35,6 +35,13 @@
 
         <!-- we expect packaging formats to have integration tests, but not unit tests -->
         <skip.unit.tests>true</skip.unit.tests>
+
+        <!-- By default we sign RPMs and DEBs using a dummy gpg directory that we check into
+             source control. Releases will override this with the real information. -->
+        <gpg.key>16E55242</gpg.key>
+        <gpg.keypath>${project.parent.basedir}/src/test/resources/dummyGpg</gpg.keypath>
+        <gpg.keyring>${gpg.keypath}/secring.gpg</gpg.keyring>
+        <gpg.passphrase>dummy</gpg.passphrase>
```

Comment:
This implies we have to override `gpg.key` (already do that), `gpg.keypath` (back to `~/.gnupg`, this is currently missing) properties in the release script

Side note: In order to not type the `gpg.passphrase` you can configure it in ~/.m2/settings.xml - I tested, and that configuration wins against this one as expected.